spark/src/main/scala/org/apache/iceberg/spark/SparkTableUtil.scala

```diff
@@ -297,5 +305,104 @@ object SparkTableUtil {
       )
     }
   }
+
+  private def buildManifest(conf: SerializableConfiguration,
+      sparkDataFiles: Seq[SparkDataFile],
+      partitionSpec: PartitionSpec,
+      basePath: String): Iterator[Manifest] = {
+    if (sparkDataFiles.isEmpty) {
+      return Seq.empty.iterator
```

Comment:
you don't need a `return` now for your current changes.

Reply:
I see... Thanks